lib/livebook/intellisense/identifier_matcher.ex

```diff
@@ -207,6 +207,48 @@ defmodule Livebook.Intellisense.IdentifierMatcher do
     match_module_function(mod, hint, ctx) ++ match_module_type(mod, hint, ctx)
   end
 
+  defp match_struct_fields_or_local_or_var(hint, ctx) do
+    case expand_struct_fields(ctx) do
+      {:ok, struct, fields} ->
+        for field <- fields, do: {:map_field, field, nil}
+
+      _ ->
+        match_local_or_var(hint, ctx)
+    end
+  end
+
+  defp expand_struct_fields(ctx) do
+    with {:ok, quoted} <- Code.Fragment.container_cursor_to_quoted(ctx.hint),
+         {aliases, pairs} <- find_struct_fields(quoted),
+         alias = Enum.join(aliases, "."),
+         mod = expand_alias(alias, ctx),
+         true <- has_struct?(mod) do
+      fields =
+        pairs
+        |> Enum.reduce(Map.from_struct(mod.__struct__), fn {key, _}, map ->
+          Map.delete(map, key)
+        end)
+        |> Map.keys()
```

Comment:
We could be propagating the default value for struct fields from here -- is there value in surfacing them in the autocompletion somehow?

Reply:
I think we can show the default value in the editor, so I would return it! @jonatanklosko ?

Reply:
Yeah, we can make the doc say "Default: `<inspected>`" or something like this.